enclave/chain_relay/src/justification.rs

```diff
@@ -233,65 +206,3 @@ where
 		Ok(route)
 	}
 }
-
-// copied
-
-/// A commit message for this chain's block type.
-pub type Commit<Block> = finality_grandpa::Commit<
-	<Block as BlockT>::Hash,
-	NumberFor<Block>,
-	AuthoritySignature,
-	AuthorityId,
->;
-
-mod communication {
```

Comment:
Can now be imported directly from substrate

Reply:
Nice, I always like it when PRs actually remove code. :D